src/index.html

```diff
@@ -11,7 +11,176 @@
+            name="name"
+            id="name"
+            class="name"
+            autocomplete="off"
+          >
+        </label>
+        <label for="age" class="label">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            id="age"
+            class="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+        <label for="dob" class="label">
+          Full date of birth:
+          <input
+            type="date"
+            name="dob"
+            class="dob"
+            id="dob"
+          >
+        </label>
+        <label for="terms" class="label">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="terms"
+            class="terms"
+            id="terms"
+            value="agree"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <label for="email" class="label">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            id="email"
+            class="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+        <label for="password" class="label">
+          Password:
+          <input
+            type="password"
+            name="password"
+            id="password"
+            class="password"
+            minlength="6"
+            maxlength="18"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <label for="radio" class="label">
+          Do you love cats?
+          <input
+            type="radio"
+            name="loveCats"
+            id="loveCatsYes"
+            class="radio"
+            value="yes"
+          >Yes
+          <input
+            type="radio"
+            name="loveCats"
+            id="loveCatsNo"
+            class="radio"
+            value="no"
+          >No
+        </label>
+        <label for="color" class="label">
+          What is your favorite color?
+          <input
+            type="color"
+            name="favColor"
+            id="color"
+            class="color"
+          >
+        </label>
+        <label for="bed-time" class="label">
+          What time do you go to bed?
+          <input
+            type="time"
+            step="2"
+            name="bedTime"
+            id="bed-time"
+            class="bed-time"
+          >
+        </label>
+        <label for="fav-car" class="label">
+          What are your favorite brand of cars?
+          <select
+            name="favCar"
+            id="fav-car"
+            class="fav-car-list"
+            multiple
+          >
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
```

Comment:
better pass lower-case string to attribute `value` - it will mitigate potential issues when working with data at back-end
```suggestion
            <option value="bmw">BMW</option>
            <option value="audi">Audi</option>
            <option value="lada">Lada</option>
```